### DISCUSSION TITLE
software dependency visualization thesis

### DISCUSSION THREAD

**Discussion by jspeed-meyers** [Open-Source-Software-Neighborhood-Watch/town-square#15] [Ideas]
@gps-iqt-labs, thought this software dependency visualization thesis using npm packages (from a Daniel German masters student) would interest you: https://dspace.library.uvic.ca/handle/1828/13458

**Comment by gps-iqt-labs** [comment]
Interesting -- thanks for flagging @jspeed-meyers. Some of these remind me of a CLI-based codebase visualization tool our colleague Charlie was tinkering with last summer (I just passed it on to him as well.)
